SQL/Archive/16.04/2016-01-20-InstrumentListOrdering.sql

```diff
@@ -0,0 +1,4 @@
+# add ordering column (tinyint) to battery
+ALTER TABLE test_battery ADD instr_order tinyint after firstVisit;
+# add ordering column (tinyint) to test_subgroups
+ALTER TABLE test_subgroups ADD group_order tinyint after Subgroup_name;
```

Comment:
This should be reflected in the schema. And a patch should be available somewhere else than in the archives.